Closes #1299: Adding documentation surrounding developing and applying updates to quickstart sites. (#1300)
az-digital/az_quickstart: CONTRIBUTING.md

## Paragraph Styles

Much of the content editor experience for pages in Quickstart consists of constructing pages using paragraph items as content building blocks. In crafting

> new paragraph types, it will often be necessary to include settings that are tracked as part of individual paragraph items that control the display of the related paragraph item. The recommended approach for including this sort of setting is with [Paragraph Behavior Plugins (https://www.drupal.org/project/paragraphs/issues/2832884).

Reviewer: ```suggestion
new paragraph types, it will often be necessary to include settings that are tracked as part of individual paragraph items that control the display of the related paragraph item. The recommended approach for including this sort of setting is with [Paragraph Behavior Plugins] (https://www.drupal.org/project/paragraphs/issues/2832884).
```

> new paragraph types, it will often be necessary to include settings that are tracked as part of individual paragraph items that control the display of the related paragraph item. The recommended approach for including this sort of setting is with [Paragraph Behavior Plugins] (https://www.drupal.org/project/paragraphs/issues/2832884).

Reviewer: ```suggestion
new paragraph types, it will often be necessary to include settings that are tracked as part of individual paragraph items that control the display of the related paragraph item. The recommended approach for including this sort of setting is with [Paragraph Behavior Plugins](https://www.drupal.org/project/paragraphs/issues/2832884).
```
Remove space

```
ddev drush en -y devel && ddev drupal site:mode dev
```

## Configuration Management and Database Updates

A question that frequently arises for Quickstart contributors is whether a change that they are making requires a database update or if configuration file changes are sufficient.

### Context

> `az-digital/az_quickstart` is a Drupal distribution that is hosted on many different environments by a multitude of teams with differing philosophies on
> change management when it comes to their Quickstart websites. This means that the Arizona Digital team should use the following guidelines when making changes to the Quickstart codebase.

Reviewer 2: ```suggestion
`az-digital/az_quickstart` is a Drupal distribution that is hosted on many different platforms by a variety of teams employing different strategies and cadences when it comes to updating their Quickstart websites. This necessitates relatively strict guidelines that the Arizona Digital team should follow when making changes to Quickstart itself.
```

```
ddev drush en -y devel && ddev drupal site:mode dev
```

### Configuration Management and Database Updates

Many times the issue of how or when to make a database update, or whether to
simply rely on configuration file changes when adding or changing quickstart
functionality comes up.
#### Context
`az-digital/az_quickstart` is a drupal distribution that is hosted on many
different environments by a multitude of teams with differing philosophies on
change management when it comes to their Quickstart websites. This means that
the Arizona Digital team should use the following guidelines when making changes
to the Quickstart codebase.

> #### When to make a database update as part of your change pull request.

Reviewer 2: ```suggestion
#### Configuration changes that require database updates
```

If your code change:
-  Adds a new setting that never existed before.
-  It would be best if the new setting is immediately set to the default
   regardless of downstream decisions.
-  Replaces an existing setting and has a new key name.

> #### When to rely on just a configuration yaml file change.

Reviewer 2: ```suggestion
#### Configuration changes that do not require database updates
```
